Library/Homebrew/os.rb

```diff
@@ -13,6 +13,14 @@ def self.linux?
     RbConfig::CONFIG["host_os"].include? "linux"
   end
 
+  def self.kernel_version
+    @kernel_version ||= if output = Utils.popen_read("uname", "-r").chomp.presence
```

Comment:
```suggestion
    @kernel_version ||= if output = Utils.safe_popen_read("uname", "-r").chomp.presence
```

I don't think this should be able to fail, right?

Reply:
Whoops, I forgot about that. I was just preserving the functionality of `OS::Linux::Kernel.version` since that's where this came from

We probably won't need the `if` since `safe_popen_read` raises an error anyways